code/HISPANIA/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/twohanded/kinetic_crusher
+	var/proyectile_type = /obj/item/projectile/destabilizer
+
+/obj/item/twohanded/kinetic_crusher/cursed
+	name = "cursed-kinetic crusher"
+	desc = "This kinetic crusher... it's alive?"
```

Comment:
```suggestion
	desc = "This kinetic crusher... It's alive?"
```
:eye:

Reply:
Como siguieres que lo ponga? no es que se me de muy bien el ingles.

Reply:
Bueno era para que pusieras la I en mayuscula pero si seria mejor cambiar la desc, ponla como:
"A red-bloodish kinetic crusher that appears to be alive."